vscode/src/testExplorer.ts

```diff
@@ -0,0 +1,267 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+// This file uses the VS Code Test Explorer API (https://code.visualstudio.com/docs/editor/testing)
+
+import * as vscode from "vscode";
+import { loadProject } from "./projectSystem";
+import {
+  getCompilerWorker,
+  ICompilerWorker,
+  IProjectConfig,
+  log,
+  ProgramConfig,
+  QscEventTarget,
+} from "qsharp-lang";
+import { getActiveQSharpDocumentUri } from "./programConfig";
+import { IProgramConfig } from "../../npm/qsharp/lib/web/qsc_wasm";
```

Comment:
You should be importing from `qsharp-lang`, not walking up and down the directory tree

Reply:
(i.e. this should be in the list imported on line 8)

Reply:
Ah yeah, LSP auto-import hit me here. Updated.